### PR TITLE
Dynamix Work section and Popup Window

### PR DESCRIPTION
**Pull request by ridaarif98**
- In an array store project information
- Embed array in HTML dynamically
- On click show popup window of the project detail
- Click on the close icon to close the popup window
- Add design in the stylesheet for the popup window
- Integrate linters
- Resolve linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -172,9 +172,41 @@ <h1>My Recent Works</h1>
...
+      and scrambled it 1960s with the releaLorem Ipsum is simply dummy text
+      of the printing and typesetting industry. Lorem Ipsum has been the
+      industry's standard dummy text ever since the 1500s,
+     when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+  <ul class="See-btn1">
+    <li><button type="button" class="see-project">See Live<i class="fa fa-external-link" aria-hidden="true"></i></button></li>
+    <li><button type="button" class="see-project">See Source<i class="fa fa-github" aria-hidden="true"></i></button></li>
+  </ul>
+</li>
+</ul> -->
```
Comment (Dipeshtwis) [inline index.html:207]: - [ ] Commenting code is not a good way when you are sending for production, it usually use for testing purpose and should deleted while sending for production. Please remove all the commented code to make your code more cleaner and nicer.

File: script/workScript.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    id: 5,
+    name: 'Multi-Post Stories Gain+Glory',
+    photo: './Images/SecondPortfolio.svg',
+    languages_list: ['Ruby on rails', 'CSS', 'Javascript', 'Html'],
+    btn_text: 'See Project',
+  },
+  {
+    id: 6,
+    name: 'Multi-Post Stories Gain+Glory',
+    photo: './Images/SecondPortfolio.svg',
```
Comment (Dipeshtwis) [inline script/workScript.js:42]: - [ ] It looks like all the data are exactly the same on the script, to use different information for each project make it easier to see bugs and have a project more like it should be work, I invite you to change the info and fix the necessary issues on your project after that.

File: script/workScript.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    </ul>
+    <div class="list-boxPop">
+    <ul>
+    ${(window.innerWidth < 768) ? languagesPopup(works.languages_list) : languagesPopupDesktop(works.desktoplist)}
+    </ul>
+    </div>
+    <p>${works.description}</p>
+  <ul class="See-btn1">
+    <li><a href=${works.liveLink}><button type="button" class="see-project">${works.firstBtn}<i class="fa fa-external-link" aria-hidden="true"></i></button></a></li>
+    <li><a href=${works.sourceLink}><button type="button" class="see-project">${works.secondBtn}<i class="fa fa-github" aria-hidden="true"></i></button></a></li>
```
Comment (Dipeshtwis) [inline script/workScript.js:118]: - [ ] In desktop verson from `1024px` to `1077px`  the description hides under the live button. Below is the screenshot for reference. Please fix that.

Screenshot from 2021-06-30 17-11-23

File: style.css [hunk 4]
```diff
@@ -294,6 +294,148 @@ body {
...
+
+.flex-box1 .firstListPop {
+  padding: 0;
+  list-style: none;
+}
+
+/*
+.list-boxPop ul li:nth-child(4), li:nth-child(5),li:nth-child(6),li:nth-child(7) {
+  display: none;
+}  */
```
Comment (Dipeshtwis) [inline style.css:338]: - [ ] Remove the commented code.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @ridaarif98, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of Flexbox. :clap: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**